protocols/virtualfund/virtualfund-single-hop_test.go

```diff
@@ -200,7 +200,7 @@ func TestSingleHopVirtualFund(t *testing.T) {
 
 		testNew := func(t *testing.T) {
 			// Assert that a valid set of constructor args does not result in an error
-			o, err := New(false, vPreFund, my.address, 2, my.role, ledgerChannelToMyLeft, ledgerChannelToMyRight)
+			o, err := New(false, vPreFund, my.address, 2, my.role, &ledgerChannelToMyLeft, ledgerChannelToMyRight)
```

Comment:
```suggestion
			o, err := New(false, vPreFund, my.address, 2, my.role, &ledgerChannelToMyLeft, &ledgerChannelToMyRight)
```

Why isn't this a compiler error??

Reply:
`ledgerChannelToMyRight` is set to the value of  `NewTwoPartyLedger` which returns a pointer. `ledgerChannelToMyLeft` is just an empty struct that we set in the test. We should probably update `ledgerChannelToMyLeft` to be a pointer to be consistent.

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/235/commits/bd2e03c1643bffaebfe52ab371e068f7ee3c9c81 so that `ledgerChannelToMyLeft` is a pointer for consistency.